website/src/pages/Section.js

```diff
@@ -78,6 +89,24 @@ const Section = ({ location, ...props }) => (
                     : null
               )}
             </Content>
+            <StyledFlex>
+              {previous && (
+                <Block>
+                  <Block>Previous article</Block>
+                  <StyledLink href={getSectionUrl(sections, previous)}>
```

Comment:
Use react-router `Link` here (not `NavLink`). Otherwise it'll trigger a browser refresh.
See https://github.com/diegohaz/reakit/blob/master/website/src/components/Menu.js#L66